luatest/runner.lua

```diff
@@ -148,6 +149,13 @@ function Runner.parse_cmd_line(args)
             result.output_file_name = next_arg()
         elseif arg == '--repeat' or arg == '-r' then
             result.exe_repeat = tonumber(next_arg()) or error('Invalid value for -r option. Integer required.')
+        elseif arg == '--group-repeat' or arg == '-g' then
```

Comment:
I like this. "repeat group" is better because it sounds like a command or action, while "group repeat" is a state-of-things. Same in Russian:

* повтори группу
* групповое повторение